node/core/bitfield-signing/src/lib.rs

```diff
@@ -268,10 +268,20 @@ impl JobTrait for BitfieldSigningJob {
 			drop(span_availability);
 			let _span = span.child("signing");
 
-			let signed_bitfield = validator
-				.sign(keystore.clone(), bitfield)
+			let signed_bitfield = match validator.sign(keystore.clone(), bitfield)
 				.await
-				.map_err(|e| Error::Keystore(e))?;
+				.map_err(|e| Error::Keystore(e))?
+			{
+				Some(b) => b,
+				None => {
+					tracing::error!(
+						target: LOG_TARGET,
+						"Key was found at construction, but while signing it could not be found.",
+					);
+					return Ok(());
```

Comment:
Is this fine? I assume that the task will stop running but the node will not go down.